<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addsales</class>
 <widget class="QDialog" name="addsales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加经办人信息</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>添加经办人信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>确认添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="addsales_Layout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
        </property>
        <property name="text">
         <string>编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_num"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="addsales_Layout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
        </property>
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="addsales_Layout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
        </property>
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
        </property>
        <property name="text">
         <string>男</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">sexgroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
        </property>
        <property name="text">
         <string>女</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">sexgroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="addsales_Layout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
        </property>
        <property name="text">
         <string>电话：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_phone"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>261</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="addsales_Layout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>备注：</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexgroup"/>
 </buttongroups>
</ui>
